spec/active_hash/base_spec.rb

```diff
@@ -594,9 +594,15 @@ class Region < ActiveHash::Base
       end
 
       it "raises ActiveHash::RecordNotFound when id not found" do
-        proc do
+        begin
```

Comment:
Seems like rspec 2.2 does not support `have_attributes` for `raise_error` so I did this rather ugly thing here. Open for better solutions though :/

```ruby
RSpec.describe "composing matchers" do
  it "raises StandardError" do
    expect { raise StandardError, "my message" }.
      to raise_error(an_instance_of(StandardError).and having_attributes({"message" => "my message"}))
  end
end
```

https://relishapp.com/rspec/rspec-expectations/v/2-2/docs/matchers/expect-error
https://relishapp.com/rspec/rspec-expectations/v/3-9/docs/built-in-matchers/raise-error-matcher#set-expectations-on-error-object-with-chained-matchers